FreeAgent.postman_collection.json

```diff
@@ -1152,6 +1152,160 @@
 				}
 			]
 		},
+		{
+			"name": "Bill Items",
+			"item": [
+				{
+					"name": "index",
```

Comment:
Nitpick, others have friendlier names so this should match and be "List Bill Items" or similar.

Reply:
I've renamed this to be a bit nicer